trec_covid_semantic_search/index.json

```diff
@@ -0,0 +1,46 @@
+{
+  "settings": {
+    "index.number_of_shards": {{number_of_shards | default(1)}},
```

Comment:
IMO, we can leave defaults to cluster. in other words, lets set this value if number_of_shards are provided in params